### DISCUSSION TITLE
Alt + drag & dropping node into script should also create onready

### DISCUSSION THREAD

**Discussion by aquast1** [godotengine/godot-proposals#6899] [Editor]
Before opening a proposal issue I'd like to open a discussion around this even though it's pretty small.
Currently the process of ctrl + drag is slightly annoying as it overlaps with the functionality of selecting multiple nodes.
If a node is already selected and you ctrl + click you'll either deselect the node you want to include in the script or include another node, it almost always requires 2 unintuitive clicks to perform an action that should be super quick.
I propose either adding "ALT" to if statement that was added in [this PR](https://github.com/godotengine/godot/pull/60708/files) (editor/plugins/script_text_editor.cpp:1562) OR making this controllable through the editor settings although I wouldn't know what to propose for that specifically

**Comment by aquast1** [comment]
To add to this, might be cool for various modifier keys to do different things but I can't think of anything specific beyond having one that declares a var without "@onready"

**Comment by theraot** [comment]
Note: You can hold Ctrl just before dropping. i.e. You don't need to have Ctrl pressed before you start dragging.

**Reply by aquast1** [reply]
Ah didn't realize that, thank you! I still hope this functionality can be improved or expanded on though.
